src/actions/account.js

```diff
@@ -147,6 +147,15 @@ export const { addAccessKey, clearAlert } = createActions({
          errorCodeDescription: '' 
       })
    ],
+   ADD_ACCESS_KEY_SEED_PHRASE: [
+      wallet.addAccessKey.bind(wallet),
+      () => ({
+         successCodeHeader: 'Success',
```

Comment:
Let's make this localizable, same as `RequestStatusBox`. I think ideally it should also use same format of `meta`, if we ever need custom headers – one way is to lookup `${messageCode}.header` message as header.